magenta/js/music_rnn/demo/index.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as magenta from '@magenta/core';
+import NoteSequence = magenta.NoteSequence;
+import INoteSequence = magenta.INoteSequence;
+import { MelodyRnn } from '../src/index';
+import * as dl from 'deeplearn';
+
+const MELODY_NS = NoteSequence.create({
```

Comment:
This would be a lot cleaner with INoteSequence since you can just pass it in arbitrary objects instead of having to do NoteSequence.blah.create